WeatherMonitot.Tests/WeatherTests/WeatherShould.cs

```diff
@@ -0,0 +1,41 @@
+﻿using AutoFixture;
+using AutoFixture.AutoMoq;
+using Moq;
+using WeatherMonitor.Bots;
+using WeatherMonitor.PrintConfig;
+using WeatherMonitor.WeatherManagement;
+namespace WeatherMonitor.Tests.WeatherTests;
+
+public class WeatherShould
+{
+    private readonly Mock<Weather> _weather;
+    private readonly WeatherData _weatherData;
+
+    public WeatherShould()
+    {
+        _weather = new Mock<Weather>() { CallBase = true }; ;
+
+        var fixture = new Fixture();
```

Comment:
declare fixture as a global variable here

Reply:
but since fixture here is used just once inside the ctor, is there a benefit of make it global ?

Reply:
Yes for future tests